telemetry/specs/INT.mdk

```diff
@@ -1220,6 +1251,128 @@ Metadata if added by the source. The total stack length can be determined
+
+* The fields in the INT metadata header are interpreted the following way:
+  - Ver (4b): INT metadata header version. Should be 2 for this version.
+  - D   (1b): Discard Copy/Clone. Not applicable in INT-MX mode. Should be set to 0 
+         by the INT Source.
+  - E   (1b): Max Hop Count exceeded. Not applicable in INT-MX mode. Should be set to 0 
+         by the INT Source.
+  - M   (1b): MTU exceeded. Not applicable in INT-MX mode. Should be set to 0 
+         by the INT Source.
+  - R  (25b): Reserved bits. Should be set to 0 by INT Source and ignored by other devices.
+      
+* INT instructions are encoded as a bitmap in the 16-bit INT Instruction field:
+each bit corresponds to a specific standard metadata as specified in Section 3.
+  - bit0 (MSB): Switch ID
+  - bit1: Level 1 Ingress Interface ID (16 bits) + Egress Interface ID (16 bits)
+  - bit2: Hop latency
+  - bit3: Queue ID (8 bits) + Queue occupancy (24 bits)
+  - bit4: Ingress timestamp (8 bytes)
+  - bit5: Egress timestamp (8 bytes)
+  - bit6: Level 2 Ingress Interface ID + Egress Interface ID (4 bytes each)
+  - bit7: Egress interface Tx utilization
+  - bit8: Buffer ID (8 bits) + Buffer occupancy (24 bits)
+  - bit15: Checksum Complement
+  - The remaining bits are reserved.
+  
+  Semantics of Queue occupancy and Buffer occupancy is the default semantics of
+  those two metadata. Additional semantics as needed for different implementation
+  can be defined in the metadata semantics YANG model. 
+  
+  Details of the metadata semantics YANG model can be accessed at the link below:
+  https://github.com/p4lang/p4-applications/blob/master/telemetry/code/models/p4-dtel-metadata-semantics.yang
+  
+  Bits 0 - 14 are Baseline INT Instructions. Each instruction bit that is set
+  requests 4 bytes of metadata be sent to monitoring system at each hop, 
+  except for bits 4-6, each requires 8 bytes of metadata. 
+  
+* Domain Specific ID (16b): the unique ID of the INT Domain.
+  
+* DS Instruction (16b): Instruction bit map specific to the INT domain identified by the 
+  Domain Specific ID. Domain Specific Instruction is an instruction that requires additional 
+  processing of Domain Specific Flags (DS Flags) for the INT Domain identified by Domain Specific ID. 
+  If the Domain Specific ID matches any Domain ID known to this node, then additional processing
+  of the Domain Specific Flags and Domain Specific Instruction is required and Domain Specific
+  Metadata is also sent to the monitoring systems.
+
+  Some metadata can be defined as 'source-only' metadata by the INT domain. Those metadata
+  will be sent only by the INT source, not by INT transit or sink devices.
+  In a sense, 'source-only' bits do not serve as instructions for downstream INT devices
+  to follow. In INT-MX mode, the source node should not set the bits corresponding to 
+  "source-only" metadata in the DS Instruction bitmap. However, the source node should ensure
```

Comment:
s/should/must
...
when such "source-only" metadata is included.

Reply:
Done.